### ISSUE TITLE
member-access rule should trigger with readonly parameter property

### ISSUE THREAD

**Issue by cyrilgandon** [palantir/tslint#3323]
### Bug Report

- __TSLint version__: 5.7.0
- __TypeScript version__: 2.5.3
- __Running TSLint via__:  VSCode

#### TypeScript code being linted

```ts
class Point {
    constructor(
        readonly x: number,
        readonly y: number
    ) { }
}
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "member-access": [true, "check-accessor"]
  }
}
```

#### Actual behavior
No Error
#### Expected behavior
It should trigger the [`member-access` rule](https://palantir.github.io/tslint/rules/member-access/):
>The class property 'x' must be marked either 'private', 'public', or 'protected' (member-access)

The same code, written differently, triggers the error:
```ts
class Point {
    readonly x: number; // <-- Error
    readonly y: number; // <-- Error
    constructor(x: number, y: number) {
        this.x = x;
        this.y = y;
    }
}
```

Maybe there is a need for a new configuration of this rule, like `check-parameter-property`, because I'm not sure that it make sense to have it under `check-accessor`.

**Comment by ajafff** [comment]
Fixed by #3325
